kieserver/image.yaml

```diff
@@ -200,12 +200,12 @@ envs:
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
     description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."  
-  - name: "KIE_KAFKA_EXT_DISABLED"
+  - name: "KIE_KAFKA_EXT_ENABLED"
     example: "true"
     description: "Disable Kafka extension"
   - name: "KIE_KAFKA_EXT_TOPICS"
     example: "event=events,book=books"
```

Comment:
is this description correct? seems to be misaligned with what we discussed today earlier.

Reply:
Is correct, here the user just add the mapping and then there https://github.com/jboss-container-images/jboss-kie-modules/pull/446/files#diff-3e7628a0c0671f1c47f2f4da279754577748d791191318fb85c3d250b1ca182cR741  we add the syprop on every topic in the array